### ISSUE TITLE
Failing in amp_analysis stage

### ISSUE THREAD

**Issue by MaestSi** [TGAC/RAMPART#27]
Although in the mass_select stage the final score for each of the assemblers run is correctly evaluated (so the file "weightings.tab" is correctly read in this stage), in rampart_out/8_amp_analysis/scores.tsv all the group scores are equal to 0.0 (also the ones for amp-stage-0, which are not equal to 0 in rampart_out/6_mass_select/scores.tsv).
Is there an issue in the reading of the "weightings.tab" file in the amp_analysis stage?
Thanks

**Comment by maplesond** [comment]
Yes the full calculation for the assembly scores for the amp stage was never implemented.  Ideally I wanted to modify the amp stage to allow for testing of multiple tools and parameters similar to the MASS stage.

Sent from my Samsung Galaxy smartphone.

-------- Original message --------
